lib/ButtonLink.js

```diff
@@ -1,6 +1,6 @@
-var React = require('react');
+var React = require('react/addons');
```

Comment:
React is not written as AMD (react-bootstrap is, for example), so we can't import a single file... We can either embed this dependency into the build, or use react-with-addons to get dependency via interface `React.addons.classSet`. I think that embedding will be OK since the file is very small.